core/src/elements/kick_bomb.rs

```diff
@@ -241,13 +260,13 @@ fn update_lit_kick_bombs(
             } else if !player_standing_left && player_sprite.flip_x {
                 body.velocity.x = -kick_velocity.x;
                 body.velocity.y = kick_velocity.y;
-            } else if kick_bomb.age >= *arm_delay {
+            } else if kick_bomb.fuse_time.finished() {
```

Comment:
This should be `arm_delay` here:

```suggestion
            } else if kick_bomb.arm_delay.finished() {
```

Reply:
Ok just updated this to arm_delay. About to commit right now.